### PR TITLE
Fixes #24303 - New hostgroup after submit path

### PR DESCRIPTION
**Pull request by sharvit**
Change the url to hostgroups_path
after creating a new hostgroup

### CODE REVIEW

**Comment by sharvit** [comment]
The `data-history-url` getting used here:
https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/host_edit.js#L153-L155

**Review by dLobatog — CHANGES_REQUESTED** [review]
@sharvit How about a unit test for this :smile: ? Thank you!

**Comment by sharvit** [comment]
Thanks for the review @dLobatog, test added

File: test/integration/hostgroup_js_test.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@ class HostgroupJSTest < IntegrationTestWithJavascript
     host = Hostgroup.where(:name => "myhostgroup1").first
     assert host
     assert_equal env.name, host.environment.name
+    assert page.has_current_path? hostgroups_path
```
Comment (boaz0) [inline test/integration/hostgroup_js_test.rb:30]: shouldn't it be `have_current_path`?
I think `has_current_path?` returns `true` or `false` while `have_current_path` returns the current URL (after waiting).

Comment (dLobatog) [reply test/integration/hostgroup_js_test.rb:30]: @ripcurld https://www.rubydoc.info/github/jnicklas/capybara/Capybara/SessionMatchers#has_current_path%3F-instance_method

I think this is fine, the assert is just verifying `has_current_path?` returns true. If I'm not wrong, you are mistaking `assert` by `assert_equal` (we're not comparing hostgroups_path to the result of `page.current_path` in the assertion)

**Review by dLobatog — APPROVED** [review]
I manually confirmed it fixes the issue too. This looks like an issue where it must have been difficult to find why is it happening, thanks @sharvit

**Pull request merged**